packages/react-devtools-shared/src/utils.js

```diff
@@ -447,6 +448,9 @@ export function getDataType(data: Object): DataType {
     case 'symbol':
       return 'symbol';
     default:
+      if (data instanceof HTMLAllCollection) {
+        return 'html_all_collection';
+      }
```

Comment:
shouldn't this be done above the switch? (like here: https://github.com/facebook/react/blob/b3da52be8915548ef36d8ac83c4611e8c58c478c/packages/react-devtools-shared/src/utils.js#L405)

Because I think that HTMLAllCollection has the type `object` so this case might not be checked
and also, like line 405, checking if `HTMLAllCollection` is defined could be good check too